transifex/transifex-javascript
# chore: Upgrade to Angular v14

__DESCRIPTION:__
Angular v13 support window ends [2023-05-04](https://angular.io/guide/releases#actively-supported-versions). Angular v16 is expected [2023-05-01](https://angular.io/guide/releases#release-schedule) (BTW [documentation](https://developers.transifex.com/docs/angular-sdk) does not express support for Angular v15 currently. But the [code](https://github.com/transifex/transifex-javascript/blob/master/packages/angular/projects/tx-native-angular-sdk/package.json#L19-L21) does. Should this be updated?)
My analysis of the current Angular code shows nothing that is a breaking change from Angular v13 to v15.

Additionally I have refreshed the dependencies and updated the lint configuration to current recommendations from Angular / @angular-eslint/eslint-plugin. All code changes are automatic formatting and fixing from tooling.

Finally I have run `npx package-json-validator --warnings --recommendations` and `npx sort-package-json`.

__STEPS TO TEST:__
1. `ng lint`
2. `ng test`
3. `ng build`

All linting and tests pass on my system. Build appears to be good as well. All of my future contributions to this library will likely be based off of these changes.
